docs/api.md

```diff
@@ -23,6 +23,11 @@ mainly, but not only, provisioning details.
         and the port is optional, if using the default one (623).
     * Dell iDRAC
       * `idrac://` (or `idrac+http://` to disable TLS).
+      * `idrac-redfish://``may be used as a prefix to select use of an iDRAC
+        controller with the Redfish protocol as configuration requets  require
+        specific handling with Dell BMCs. The URL must also contain a path to
+        the system ID, for example
+        `idrac-redfish://myhost.example/redfish/v1/Systems/MySystemExample`
```

Comment:
It would be good to be consistent with #413 here

```suggestion
        `idrac-redfish://myhost.example/redfish/v1/Systems/System.Embedded.1`
        or `idrac-redfish://myhost.example/redfish/v1/Systems/1`
```

Reply:
I've got the actual default location in another window, I suspect that would be best....